libr/anal/p/anal_arm_cs.c

```diff
@@ -2,10 +2,14 @@
 
 #include <r_anal.h>
 #include <r_lib.h>
+#include <ht_uu.h>
+#include <capstone/capstone.h>
```

Comment:
shouldnt this be just capstone.h after rebase?

Reply:
yes
```suggestion
#include <capstone.h>
```